components/raftstore/src/store/worker/pd.rs

```diff
@@ -214,6 +216,43 @@ pub struct PeerStat {
     pub approximate_size: u64,
 }
 
+#[derive(Default)]
+pub struct PeerReportReadStat {
+    pub region_id: u64,
+    pub report_read_bytes: u64,
+    pub report_read_keys: u64,
+    pub report_query_stats: QueryStats,
+}
+
+impl Ord for PeerReportReadStat {
+    fn cmp(&self, other: &Self) -> CmpOrdering {
+        match self.report_read_bytes.cmp(&other.report_read_bytes) {
+            CmpOrdering::Equal => self.report_read_keys.cmp(&other.report_read_keys),
```

Comment:
only cmp byte and key? if a region with high qps and low byte, it may be filtered.

Reply:
What's your advice to implement `Ord` for `QueryStats`?

Reply:
In the previous logic, the highest dimension of any one of the dimensions would definitely be selected and sent to pd, but the current logic requires that all three dimensions in order be relatively high before they are sent to pd, which will reduce the search space of pd.

Reply:
yep, I think this is a trade off. We reduce the search space in order to stable the store heartbeat.